dashboard/views.py

```diff
@@ -6,7 +7,19 @@
 from push.models import PushApplication
 
 
-class Home(TemplateView):
+class Login(TemplateView):
+    template_name = 'dashboard/login.html'
+
+
+class Status(TemplateView):
```

Comment:
Sure. I like the idea of having a separate */push* page for the form. That way, the page either loads completely or not at all depending on whether someone is logged in. We could even add an *Add push application* button to the main navigation to take people there.